src/app/signUp/page.tsx

```diff
@@ -34,6 +37,43 @@ export default function SignUpPage() {
     console.log(data);
   };
 
+  const handleSignUp = (): any => {
+    const email = getValues("email");
+    const password = getValues("password");
+    const name = getValues("name");
+    const employment = getValues("employment");
+    const degree = getValues("studyingDegree");
+    if (
+      degree === "" ||
+      degree === undefined ||
+      selectedEducation === "" ||
+      selectedEducation === undefined ||
+      lookingForWork === undefined ||
+      name === "" ||
+      name === undefined ||
+      employment === "" ||
+      employment === undefined
+    ) {
```

Comment:
Can simplify to 

`if (!degree || !selectedEducation ...)` etc

Reply:
Empty string and undefined evaluate both to false

Reply:
Yup, will do